.gitlab-ci/build.yml

```diff
@@ -1,16 +1,39 @@
 ---
 pipeline image:
   stage: build
-  image: docker:20.10.22-cli
+  image:
+    name: gcr.io/kaniko-project/executor:v1.9.2-debug
+    entrypoint: [""]
   variables:
-    DOCKER_TLS_CERTDIR: ""
-  services:
-    - name: docker:20.10.22-dind
-      # See https://gitlab.com/gitlab-org/gitlab-runner/-/issues/27300 for why this is required
-      command: ["--tls=false"]
+    CACHE_TTL: 730h
   before_script:
-    - echo $CI_REGISTRY_PASSWORD | docker login -u $CI_REGISTRY_USER --password-stdin $CI_REGISTRY
+    - ""
```

Comment:
This should probably be removed

Reply:
We're pretty much back to where we started with this. There's a default `before_script` which does a rebase, among other things, which we overwrite with our own `before_script`. Since `rootless` does not contain Bash, the rebase script will not work. I'm ok with this, since I think it's highly unlikely that we'll need a rebase for the pipeline image, but I'll let you make the call. If we feel need for the `rebase.sh` to run, then we'll use a different buildkit image and go from there.

Reply:
Ah yeah well, I meant that on the version with an empty string this looks weird but with something actually there it's totally fine!